demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1103,6 +1103,45 @@ def test_is_there_spaces_in_the_end_of_name(self, pack: Pack, answer, dashboard_
         res_validator = IntegrationValidator(structure)
         assert res_validator.is_there_spaces_in_the_end_of_name() is answer
 
+    def test_ignore_test_doc_non_pack_file_should_not_ignore(self):
```

Comment:
for all tests, add more testing cases (you can use `@parametrize`)

Reply:
Fixed